.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Report a bug report to help us improve the module.
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+### Version
+{{ name }}: <!-- ex: 5.9.0 -->
+nuxt: <!-- ex: 2.0.0 -->
+
+### Nuxt configuration
+#### [mode](https://nuxtjs.org/api/configuration-mode): <!--universal is the default -->
+ - [ ] universal
+ - [ ] spa
+
+### Nuxt configuration for {{ name }}
+<!--
+    If relevant, please include the configuration you are using for this module.
+    For example:
+```
+```
+-->
+
+### Reproduction Link
+<!-- A minimal test case on https://template.nuxtjs.org/ or GitHub reprository that can reproduce the bug. -->
```

Comment:
```suggestion
<!-- A minimal test case on https://template.nuxtjs.org/ or GitHub reprository that can reproduce the bug. -->
```

Maybe it'd be good to:

* state that a reproduction is very important
* jsfiddle/jsbin/codepen is no proper reproduction

Have seen these issues mostly regarding reproduction links/repos

Reply:
Sounds good, could you update it please?

Reply:
Ok, I just updated the bug report.